src/Network-MIME/MIMEDocument.class.st

```diff
@@ -277,7 +277,8 @@ MIMEDocument >> parts [
 	[ parseStream atEnd ] whileFalse: 
 		[ messages add: msgStream upToEnd.
 		msgStream skipThisLine ].
-	^ messages collect: [ :e | MailMessage from: e ]
+	"MailMessage is not present in a minimal image"
+	^ messages collect: [ :e | (self environment at: #MailMessage) from: e ]
```

Comment:
It looks like you are trying to fix dependency issue here.

I don't know details of MIMEDocument. But the method #parts looks like required part of it and it strictly depends on MailMessage. Thus we should fix dependency to have this class in minimal image or remove MIMEDocument from it. 
Otherwise you just hide the issue.

Reply:
Actually, no, I'm not trying to fix a dependency. I'm trying to fix a compile error (that is currently hidden). Is it necessary to resolve that more difficult dependency issue now, or can we let this PR fix the compile error and #3998 deal with the dependency at such time as someone who knows more than I do has time to address it?